server/src/server.ts

```diff
@@ -123,25 +123,23 @@ export default function startServer(): void {
     /**
      * Event
      */
-    app.get('/api/v2/events', userIsAuthorized, (req, res) => getEvents(req, res));
-    app.get('/api/v2/events/:id', userIsAuthorized, (req, res) => getEvent(req, res));
+    app.get('/api/v2/events', (req, res) => getEvents(req, res)); //Sehen darf jeder
```

Comment:
Wir machen aktuell keine Unterscheidung - wir wollten das auf ein Bestimmtes Team oder öffentlich machen können. 
Jede Veranstaltung öffentlich fände ich nicht so gut
Was sagst du dazu?

Reply:
Ich lass es mal auf komplett öffentlich.
Problematisch ist halt das mit dem Team, weil wir dann jemandem, der über nen access_code kommt automatisch das team assignen müssten.

Reply:
Werd ich in #159 behandeln